packages/react-components/react-utilities/src/compose/types.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { SLOT_ELEMENT_TYPE_SYMBOL, SLOT_RENDER_FUNCTION_SYMBOL } from './constants';
+import { SLOT_ELEMENT_TYPE_SYMBOL, SLOT_RENDER_FUNCTION_SYMBOL, SLOT_USER_CLASS_NAME_SYMBOL } from './constants';
```

Comment:
Consider naming this to match up with the overrideSlotClasses - maybe SLOT_ORIGINAL_CLASS_NAME_SYMBOL, SLOT_OVERRIDE_CLASS_NAME_SYMBOL, SLOT_EXTERNAL_CLASS_NAME_SYMBOL or SLOT_CALLER_CLASS_NAME_SYMBOL.  User is kind of a confusing term because people might wonder if we mean the end user.

Reply:
Renamed to `SLOT_ORIGINAL_CLASS_NAME_SYMBOL`